geocoding/lib/geocoding.dart

```diff
@@ -15,13 +15,12 @@ export 'package:geocoding_platform_interface/geocoding_platform_interface.dart';
 /// When not supplied the currently active locale of the device will be used.
 /// The `localeIdentifier` should be formatted using the syntax:
 /// [languageCode]_[countryCode] (eg. en_US or nl_NL).
-Future<List<Location>> locationFromAddress(
+Future<List<Location>>? locationFromAddress(
   String address, {
   String? localeIdentifier,
 }) =>
-    GeocodingPlatform.instance.locationFromAddress(
+    GeocodingPlatform.instance?.locationFromAddress(
```

Comment:
In line with the remark above, I think it is better to use the bang (`!`) operator to enforce the `GeocodingPlatform.instance` not to be `null`:

```suggestion
    GeocodingPlatform.instance!.locationFromAddress(
```